tests/integration/cli/test_entry.py

```
@@ -47,12 +47,15 @@ def _project(tmp_path, monkeypatch) -> None:
 
 @pytest.mark.usefixtures("_project")
 class TestEntry:
+    # Bare `vekna` is the daemon and blocks, so what is checked here is that
+    # the tree is reachable — the daemon itself is exercised in test_daemon.py.
     @staticmethod
-    def test_the_bare_command_prints_its_help():
-        result = CliRunner().invoke(init_command(), [])
+    def test_the_help_lists_the_commands():
```

Ruff (0.16.1)

[warning] Missing return type annotation for staticmethod `test_the_help_lists_the_commands`

Add return type annotation: `None`

(ANN205)